.github/workflows/test_pull_requests.yml

```
@@ -125,7 +125,7 @@ jobs:
             backend: pyqt6
             coverage: no_cov
           - python: "3.12"
-            platform: macos-latest
+            platform: macos-12
```

```suggestion
            platform: macos-13
```

13 should be also x86-64? Or you tested it?

---

macOS-13 has x64 runner, but the question is what is the default?
macOS-14 has clearly switched to arm64 default, but also has x64.

---

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

based on this information, the macos-13 means x86 runner and macos-14/macos-latest means arm runner.

---

Checking the GitHub runner-images repo (https://github.com/actions/runner-images?tab=readme-ov-file#available-images) I think `macos-13` means x86 indeed :+1: